src/create-store.js

```diff
@@ -6,9 +6,27 @@
 
 import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
-import { createLogger } from 'redux-logger';
 import reducers from './reducers';
-import type { Store } from './types/store';
+import type { Store, Action } from './types/store';
+
+/**
+ * Create a more minimalist action logger.
+ */
+export const logger = (store: Store) => (next: Action => mixed) => (
+  action: Action
+) => {
+  const prevState = store.getState();
+  const result = next(action);
+  const nextState = store.getState();
+  const style = 'font-weight: bold; color: #0af';
+  console.log(`[action]    %c${action.type}`, style, {
+    action,
+    prevState,
+    nextState,
+    callStack: new Error().stack.split('\n').map(line => line.split('@')[0]),
+  });
+  return result;
+};
```

Comment:
I get a similar output, (and better: diff and duration !) with redux-logger using: 
```javascript
      createLogger({
        collapsed: true,
        titleFormatter: (action, time, duration) =>
          `[action]    ${action.type} (in ${duration} ms)`,
        logErrors: false,
        diff: true,
        duration: true,
      })
```